install/Create_System_Configuration_Using_Cluster_Discovery_Service.md

```diff
@@ -0,0 +1,400 @@
+   ```
+
+1. (`pit#`) Collect the CMC inventory data.
+
+   ```bash
+   "${CVT_PATH}"/cds/pdu_cmc_inventory.py --cmc_file cmclist
+   ```
+
+#### 3.4.2 SHCD data population
+
+1. Copy the SHCD files to the present working directory.
+
+1. (`pit#`) Create a JSON file using the CANU tool.
+
+   See [Validate the SHCD](../operations/network/management_network/validate_shcd.md#validate-the-shcd).
+
+   Example command:
+
+   ```bash
+   canu validate shcd --shcd SHCD.xlsx --architecture V1 --tabs 40G_10G,NMN,HMN --corners I12,Q38,I13,Q21,J20,U38 --json --out cabling.json
+   ```
+
+1. (`pit#`) Store the SHCD data in the CVT database.
+
+   ```bash
+   "${CVT_PATH}"/parse_shcd.py --canu_json_file cabling.json
+   ```
+
+#### 3.4.3 Server classification
+
+1. (`pit#`) Classify the servers.
+
+   > In the following command, replace `<N>` with the number of worker NCNs in the system.
+
+   ```bash
+   "${CVT_PATH}"/scripts/server_classification.py --no_of_workers <N>
+   ```
+
+#### 3.4.4 Compare the SHCD data with CVT inventory data
+
+1. (`pit#`) List the snapshot IDs of the SHCD and the CVT inventory data.
+
+   ```bash
+   "${CVT_PATH}"/shcd_compare.py --list
+   ```
+
+1. (`pit#`) Compare the SHCD data with the CVT inventory data.
+
+   ```bash
+   "${CVT_PATH}"/shcd_compare.py --shcd_id <SnapshotID_SHCD> --cvt_id <SnapshotID_CVT>
+   ```
+
+   Example command:
+
+   ```bash
+   "${CVT_PATH}"/shcd_compare.py --shcd_id 4ea86f99-47ca-4cd9-b142-ea572d0566bf --cvt_id 326a6edd-adce-4d1f-9f30-ae9539284733
+   ```
+
+## 4. Generate seed files and paddle file
+
+1. (`pit#`) Generate the following seed files: `switch_metadata.csv`, `application_node_config.yaml`, `hmn_connections.json`, and `ncn_metadata.csv`.
+
+   ```bash
+   "${CVT_PATH}"/create_seedfile.py
+   ```
+
+   > **NOTE**: The generated seed files will be stored in the `seedfiles` folder in the working directory.
+
+1. (`pit#`) Generate paddle file: `cvt-ccj.json`.
+
+   ```bash
+   "${CVT_PATH}"/create_paddle_file.py --architecture <arch_type>
+   ```
+
+   Where, the variable `<arch_type>` describes the type of architecture (FULL, V1, or TDS).
+   See [Generate configuration files](../operations/network/management_network/generate_switch_configs.md#generate-configuration-files) for the characteristics of the architecture.
+
+1. Copy the generated paddle file and seed files to the `prep` directory.
+
+   ```bash
+   cp -r ${PITDATA}/prep/cds/cvt-ccj.json ${PITDATA}/prep/cds/seedfiles/* ${PITDATA}/prep/
+   ```
+
+1. Create `cabinets.yaml` file in the `prep` directory, see [Create `cabinets.yaml`](create_cabinets_yaml.md).
+
+## 5. Stop discovery
+
+1. (`pit#`) Clean and reset the modified files to their original default state.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_stop.py -u bmc_username -p bmc_password
+   ```
+
+1. (`pit#`) Change into the `prep` directory.
+
+   ```bash
+   cd "${PITDATA}/prep"
+   ```
+
+## 6. Run CSI
+
+1. (`pit#`) Generate the initial configuration for CSI.
```

Comment:
The change directory step makes more sense as the first step in section 6 than as the final step in section 5.
````suggestion
(`pit#`) Clean and reset the modified files to their original default state.

```bash
"${CVT_PATH}"/cds/discover_stop.py -u bmc_username -p bmc_password
```

## 6. Run CSI

1. (`pit#`) Change into the `prep` directory.

   ```bash
   cd "${PITDATA}/prep"
   ```

1. (`pit#`) Generate the initial configuration for CSI.
````